source/common/network/io_socket_error_impl.h

```diff
@@ -29,8 +35,15 @@ class IoSocketError : public Api::IoError {
   // Deallocate memory only if the error is not Again.
   static void deleteIoError(Api::IoError* err);
 
+  // Map a system error code to an Envoy `IoErrorCode`
+  static Api::IoError::IoErrorCode errorCodeFromErrno(int sys_errno);
```

Comment:
nit: I think this method could also be private.

Reply:
I guess that it might be useful functionality in general. I guess if anyone wants to use it, they can make it public.

Reply:
fixed in 5d4e5a465e9e34df1ddbdbd8b9c55cd24e675e30